tests/table/test_count.py

```diff
@@ -1,11 +1,19 @@
-from tests.base import DBTestCase
+from tests.base import DBTestCase, engines_only
 from tests.example_apps.music.tables import Band
 
 
 class TestCount(DBTestCase):
     def test_exists(self):
         self.insert_rows()
 
-        response = Band.count().where(Band.name == "Pythonistas").run_sync()
+        response = Band.count().where(Band.popularity == 10).run_sync()
 
-        self.assertEqual(response, 1)
+        self.assertEqual(response, 2)
+
+    @engines_only("postgres", "cockroach")
```

Comment:
You should remove this line because with your new updates `count distinct `also works in `Sqlite` :)

Reply:
Good point - changed.